docs/language/ql-training/java/intro-ql-java.rst

```diff
@@ -134,15 +130,15 @@ A predicate allows you to pull out and name parts of a query.
 
 .. note::
 
-   A `QL predicate <https://help.semmle.com/QL/ql-handbook/predicates.html>`__ takes zero or more parameters, and its body is a condition on those parameters. The predicate may (or may not) hold. Predicates may also be `recursive <https://help.semmle.com/QL/ql-handbook/predicates.html#recursive-predicates>`__, simply by referring to themselves (directly or indirectly).
+   A `predicate <https://help.semmle.com/QL/ql-handbook/predicates.html>`__ takes zero or more parameters, and its body is a condition on those parameters. The predicate may (or may not) hold. Predicates may also be `recursive <https://help.semmle.com/QL/ql-handbook/predicates.html#recursive-predicates>`__, simply by referring to themselves (directly or indirectly).
 
    You can imagine a predicate to be a self-contained from-where-select statement, that produces an intermediate relation, or table. In this case, the ``isEmpty`` predicate will be the set of all blocks which are empty.
 
 
-Classes in QL
-=============
+Classes
+=======
 
-A QL class allows you to name a set of values and define (member) predicates on them.
+A class allows you to name a set of values and define (member) predicates on them.
```

Comment:
Again, I think it's worth retaining the QL here, if we can.

Reply:
Yep. It should probably be CodeQL class too.

Reply:
Hmm, actually I'll leave as QL class for now as there is no mention of any code here.